bigbluebutton-html5/imports/ui/components/polling/styles.scss

```diff
@@ -8,6 +8,7 @@
   display: flex;
   flex-direction: row;
   justify-content: center;
+  flex-grow: 0;
```

Comment:
May I ask what this is fixing?
The width is already 100%. Not that you prevented it from growing..